vowpalwabbit/offset_tree_internal.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+#include <cstdint>
+#include <vector>
+#include "learner.h"
+namespace offset_tree
```

Comment:
This should be nested inside of the VW namespace.

Side comment: I would really like the see the base namespace be lowercase `vw`. Uppercase seems very out of place with everything else. But `vw` is already taken by the global object so we would need to rename it (probably not a bad idea)